src/modules/Home/actions/home.ts

```diff
@@ -0,0 +1,14 @@
+import {createAction} from 'redux-actions';
+import {CHANGE_MAIN_SLIDE, DISANIMATE_MAIN_SLIDER, SET_MAIN_SLIDE} from '../constants/home';
+
+export const changeCurrentMainSlide = createAction(
+	CHANGE_MAIN_SLIDE,
+	(index: number) => index,
+);
+
+export const disanimateMainSlider = createAction(DISANIMATE_MAIN_SLIDER);
+
+export const setCurrentMainSlide = createAction(
+	SET_MAIN_SLIDE,
+	(index: number) => index,
```

Comment:
do not use index at actions. indexes unsafe way to work with data.

Reply:
Why? It works perfectly fine in this case

Reply:
make this logic in a slider component in a state, not in store

Reply:
done